test-result-summary-client/src/Build/PossibleIssues.jsx

```diff
@@ -15,6 +17,29 @@ export default class PossibleIssues extends Component {
         await this.fetchIssues();
     }
 
+    getUserFeedback = async(repoName, buildName, issueName, issueCreator, accuracy) => {
+        if (accuracy === true) {
+
+            const queryForFeedbackUrl = "/api/getFeedbackUrl";
```

Comment:
when calling api, the parameter values should be added too, you can refer this code with changing the parameters. https://github.com/adoptium/aqa-test-tools/blob/151d0e006ca4425d67d4436c5ff6ebe869f05676/test-result-summary-client/src/Dashboard/Widgets/Graph/Dacapo.jsx#L40